### PR TITLE
Add CSS styling for DITAVAL style

### PR DESCRIPTION
**Pull request by jelovirt**
Add CSS styling for DITAVAL `@style`. Feature is required to complete #3463

### CODE REVIEW

**Comment by infotexture** [comment]
Looks like this implements the suggestions from https://github.com/dita-ot/dita-ot/issues/3463#issuecomment-599193921 and https://github.com/dita-ot/dita-ot/pull/3482#issuecomment-605336264 to generate new default `common*.css` rules for the 5 `@style` tokens mandated by the DITA spec _(instead of outputting them to inline HTML styles)_.

@jelovirt Several questions remain:

1. How are unsupported style tokens handled?
2. Are other `<revprop>` attributes like `@color` & `@backcolor` still output as inline HTML styles?
3. How is the `@changebar` attribute handled?

File: src/main/plugins/org.dita.html5/xsl/hi-d.xsl [hunk 1]
```diff
@@ -35,9 +35,7 @@ See the accompanying LICENSE file for applicable license.
   
     <xsl:template match="*[contains(@class,' hi-d/tt ')]" name="topic.hi-d.tt">
       <span>
-        <xsl:call-template name="commonattributes">
```
Comment (jelovirt) [inline src/main/plugins/org.dita.html5/xsl/hi-d.xsl]: Fix for previous PR

File: src/main/plugins/org.dita.html5/xsl/map2html5-coverImpl_template.xsl [hunk 2]
```diff
@@ -132,7 +132,7 @@ See the accompanying LICENSE file for applicable license.
   
   <xsl:template match="*[contains(@class, ' map/map ')]" mode="chapterBody">
     <body>
-      <xsl:apply-templates select="*[contains(@class, ' ditaot-d/ditaval-startprop ')]/@outputclass" mode="add-ditaval-style"/>
+      <xsl:apply-templates select="*[contains(@class, ' ditaot-d/ditaval-startprop ')]/@style" mode="add-ditaval-style"/>
```
Comment (jelovirt) [inline src/main/plugins/org.dita.html5/xsl/map2html5-coverImpl_template.xsl]: Fix for previous PR

File: src/test/resources/bookmap1/exp/xhtml/p.html [hunk 3]
```diff
@@ -21,7 +21,7 @@ <h1 class="title topictitle1" id="ariaid-title1">New element &lt;data&gt;</h1>
 
 <div class="body conbody">
 <div class="p">In DITA 1.1, you can use new element, &lt;data&gt;. This element and the content
-inside it is ignored in the transformation process of DITA files.<div class="note note"><span class="notetitle">Note:</span> <ul class="ul">
+inside it is ignored in the transformation process of DITA files.<div class="note"><span class="notetitle">Note:</span> <ul class="ul">
```
Comment (jelovirt) [inline src/test/resources/bookmap1/exp/xhtml/p.html]: Removing duplicate class values should have been fixed ages ago.

**Comment by jelovirt** [comment]
>  How are unsupported style tokens handled?

We only support those listed in the spec. If you use something else, you have to manually add `flag__style--<token>` CSS rule yourself.

> Are other <revprop> attributes like @color & @backcolor still output as inline HTML styles?

Yes.

> How is the @changebar attribute handled?

Nothing has changed there.

**Pull request merged**